IBM-Blockchain/generator-fabric
# Use Implicit collections for private contracts instead creating one via collections.json

### Summary
* Modified the generated private contracts to use implicit collections by default rather than creating one through the `collections.json` like previously (keeping the file as an empty array though)
* Added a function to all languages (JavaScript, TypeScript, Java and Go to get the MSPID for the collection)
* For the `verifyMyPrivateAsset` function, the collection name (`mspid`) of the organisation will need to be passed in. This enhancement means that multiple collections can be used for private contracts (previously it was hardcoded to the generated collection).
* Updated the tests for all languages

### Testing
Tested all languages using the following test:

1. Test ID "001" exists in Org1 - should be false
2. Create new entry in Org1 - ID "001", Transient Data: `{"privateValue":"125"}`
3. Test ID "001" exists in Org1 - should be true
4. Read that entry in Org1
5. Update that entry in Org1 - `{"privateValue":"130"}`
6. Read that entry in Org1
7. Test ID "001" exists in Org2 - should be false
8. Verify that "001" exists in Org1 with value `{"privateValue":"130"}` `Org1MSP` - should be true
9. Delete "001" in Org1
10. Test that "001" exists in Org1 - should be false
11. Verify that "001" exists in Org1 from Org2
